### PR TITLE
Better 'What is open source?' explanation in projects page

### LINKED ISSUE #469: What is open source, explanation
### Context
Content explaining open source might be too long, or maybe confusing here: https://science.mozilla.org/projects

screen shot 2016-05-28 at 12 43 42 pm

### Expected
I guess it would be nice if the text matched, and if the open source description was simpler, now that I'm seeing it in context with the other callouts in that section:

> Open source describes software that can be used, modified, distributed, and is collaborative in nature. The extent of its "openness" is determined by the license attached to the software.

### Current

The current one is missing some commas and is a little long:

> Open source describes software that can be used modified and distributed. The extent of its "openness" is determined by the license attached to the software, but in general, open source projects are collaborative in nature, with many people contributing to the source code.

### PR DESCRIPTION
**Pull request by xuanxu**
New explanation is shorter and simpler.
Closes #469

### CODE REVIEW

**Comment by acabunoc** [comment]
For easier review

New copy:
"Open source describes software that can be used, modified, distributed, and is collaborative in nature. The extent of its "openness" is determined by the license attached to the software."
screen shot 2016-06-30 at 17 54 36



Old copy:
"Open source describes software that can be used modified and distributed. The extent of its "openness" is determined by the license attached to the software, but in general, open source projects are collaborative in nature, with many people contributing to the source code."

screen shot 2016-06-30 at 17 48 45

**Comment by acabunoc** [comment]
Flagging @auremoser

**Comment by auremoser** [comment]
Thanks @acabunoc

File: app/react/pages/projects/projects.jsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ export default React.createClass({
         <div className="threeUp-wrapper">
           <div className="container-dynamic p-t-3">
             <ThreeUp
-              item1={{ title: `What is open source?`, text: `Open source describes software that can be used modified and distributed. The extent of its "openness" is determined by the license attached to the software, but in general, open source projects are collaborative in nature, with many people contributing to the source code.`, icon: `/assets/img/icon-opensource.svg` }}
+              item1={{ title: `What is open source?`, text: `Open source describes software that can be used, modified, distributed, and is collaborative in nature. The extent of its "openness" is determined by the license attached to the software.`, icon: `/assets/img/icon-opensource.svg` }}
```
Comment (auremoser) [inline app/react/pages/projects/projects.jsx]: Looks great, merging!

**Pull request merged**